packages/node/package.json

```diff
@@ -59,6 +59,7 @@
     "pluralize": "8.0.0",
     "postcss": "^8.4.31",
     "preferred-pm": "^3.0.3",
+    "prettier": "^2.8.8",
```

Comment:
Can we use the latest version of prettier?

Reply:
we could but I'd prefer updating it a later point, at the same time as the other prettier usages across the monorepo


<img width="248" alt="Screenshot 2023-11-03 at 13 49 09" src="https://github.com/chakra-ui/panda/assets/47224540/2bd647a1-fe45-4c40-8db7-e6249e88d3d2">